src/core/Akka.Remote/Serialization/LruBoundedCache.cs

```diff
@@ -117,7 +150,7 @@ public CacheStatistics(int entries, int maxProbeDistance, double averageProbeDis
     /// <typeparam name="TValue">The type of value used in the cache.</typeparam>
     internal abstract class LruBoundedCache<TKey, TValue> where TValue : class
     {
-        protected LruBoundedCache(int capacity, int evictAgeThreshold)
+        protected LruBoundedCache(int capacity, int evictAgeThreshold, IEqualityComparer<TKey> keyComparer)
```

Comment:
I understand - I thought this is why the cache simply called an external method implemented elsewhere to do the computation via its inheritance structure? As far as I know we never used delegates in there.

Reply:
Yes, to avoid delegates is good.
But to restrict the cache to use a single/static method is not optimal.
The user of the cache has many times more context on the value creation.